e2e/bug-12629/phpstan-baseline.neon

```diff
@@ -0,0 +1,31 @@
+parameters:
+	ignoreErrors:
+		-
+			message: '#^Method Bug12629\\Bug12629\:\:is_macintosh_enc\(\) has no return type specified\.$#'
+			identifier: missingType.return
+			count: 1
+			path: src/bug-12629.php
+
+		-
+			message: '#^Method Bug12629\\Bug12629\:\:is_macintosh_enc\(\) has parameter \$s with no type specified\.$#'
+			identifier: missingType.parameter
+			count: 1
+			path: src/bug-12629.php
+
+		-
+			message: '#^Method Bug12629\\Bug12629\:\:is_macintosh_enc\(\) is unused\.$#'
+			identifier: method.unused
+			count: 1
+			path: src/bug-12629.php
+
+		-
+			message: '#^Regex pattern is invalid\: Compilation failed\: UTF\-8 error\: byte 2 top bits not 0x80 at offset 0 in pattern$#'
```

Comment:
before this fix this error message had the invalid pattern appended, which made the parser crash while loading the baseline later on